Sources/Workspace/InitPackage.swift

```diff
@@ -102,6 +146,31 @@ public final class InitPackage {
                     name: "\(pkgname)"
                 """)
 
+            var platformsParams = [String]()
+            for supportedPlatform in options.platforms {
+                let version = supportedPlatform.version
+                let platform = supportedPlatform.platform
+
+                var param = ".\(platform.manifestName)("
+                if supportedPlatform.isManifestAPIAvailable {
+                    if platform == .macOS {
+                        param += ".v10_\(version.minor)"
```

Comment:
I really dislike hardcoding parts of the version number like this, can you please just generalize this:

    if version.minor > 0 {
        param += ".v\(version.major)_\(version.minor)"
    } else {
        param += ".v\(version.major)"
    }

(This would also hypothetically work for other platforms like Windows where "minor" versions can indicate "major" releases).

Reply:
I don't want to generalize it prematurely. It'll be trivial to address this in the future as the manifest API evolves.